### PR TITLE
feat: use the latest helm chart if don't specify the version argument

### PR DESCRIPTION
**Pull request by zyy17**

### CODE REVIEW

File: cmd/app/cluster/create/create.go [hunk 1]
```diff
@@ -141,10 +141,10 @@ func NewCreateClusterCommand(l log.Logger) *cobra.Command {
 	cmd.Flags().StringVarP(&options.Namespace, "namespace", "n", "default", "Namespace of GreptimeDB cluster.")
 	cmd.Flags().BoolVar(&options.DryRun, "dry-run", false, "Output the manifests without applying them.")
 	cmd.Flags().IntVar(&options.Timeout, "timeout", -1, "Timeout in seconds for the command to complete, default is no timeout.")
-	cmd.Flags().StringVar(&options.GreptimeDBVersion, "version", manager.DefaultGreptimeDBChartVersion, "The GreptimeDB version.")
-	cmd.Flags().StringVar(&options.OperatorVersion, "operator-version", manager.DefaultGreptimeDBOperatorChartVersion, "The greptimedb-operator version.")
+	cmd.Flags().StringVar(&options.GreptimeDBChartVersion, "greptimedb-chart-version", "", "The greptimedb helm chart version, use latest version if not specified.")
+	cmd.Flags().StringVar(&options.GreptimeDBOperatorChartVersion, "greptimedb-operator-version", "", "The greptimedb-operator helm chart version, use latest version if not specified.")
```
Comment (daviderli614) [inline cmd/app/cluster/create/create.go]: The operator flag parameter is "greptimedb-operator-chart-version"

File: pkg/manager/manager.go [hunk 2]
```diff
@@ -334,3 +335,25 @@ func (m *manager) generateEtcdValues(options *CreateEtcdOptions) (map[string]int
 
 	return nil, nil
 }
+
+func (m *manager) chartDownloadURL(chartName, version string) (string, error) {
```
Comment (daviderli614) [inline pkg/manager/manager.go]: The function name is getChartDownloadURL will be better

**Comment by zyy17** [comment]
@daviderli614 PTAL

**Review by daviderli614 — APPROVED** [review]
LGTM

**Pull request merged**